ang/civicase/CaseDetailsPeople.js

```diff
@@ -23,113 +43,85 @@
     $scope.relationsAlphaFilter = '';
     $scope.relationsSelectionMode = '';
     $scope.relationsSelectedTask = '';
-    $scope.letters = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".split("");
+    $scope.letters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'.split('');
     $scope.contactTasks = CRM.civicase.contactTasks;
     $scope.ceil = Math.ceil;
-
-    function formatRole (role) {
-      var relType = relTypesByName[role.name];
-      role.role = relType.label_b_a;
-      role.contact_type = relType.contact_type_b;
-      role.contact_sub_type = relType.contact_sub_type_b;
-      role.description = (role.manager ? (ts('Case Manager.') + ' ') : '') + (relType.description || '');
-      role.relationship_type_id = relType.id;
-    }
     $scope.allRoles = [];
+    $scope.formatDate = DateHelper.formatDate;
+    $scope.getCaseRoles = getCaseRoles;
```

Comment:
add a space between properties and method definitions.

Reply:
I think these should be defined with their functions to keep consistency:

```js
$scope. getCaseRoles = function () {
  // ...
};
```

Or is there something I'm missing?

Reply:
`getCaseRoles ` is being used at other places in the controller. If define this function directly to the `$scope.getCaseRoles` then we have to order the code in such a way that other function can use these variables only after they are defined. So have followed this approach. Please feel free to ping me in case of a confusion